bauhaus/workflows/secondary/hqMetrics.py

```diff
@@ -16,66 +17,135 @@ def conditionTableType():
 
     def generate(self, pflow, ct):
         outputDict = {}
-        pflow.bundleScript("R/hqrf_pbbamr.R")
+        rscript = "R/hqrf_pbbamr.R"
+        pflow.bundleScript(rscript)
+        controls = "fasta/600bp_Control_c2.fasta"
+        pflow.bundleScript(controls)
```

Comment:
OK, that's not as it should be.  I can accept your code as is but we should open a bug that PAWS should be giving us the controls fasta in the run dir.